### DISCUSSION TITLE
Adapters Library Release & EMNLP 2023 Demo

### DISCUSSION THREAD

**Discussion by calpt** [adapter-hub/adapters#620] [Announcements]
### Initial _Adapters_ library release:

Below you find all relevant links for the _Adapters_ library release.

- Blog Post: https://adapterhub.ml/blog/2023/11/introducing-adapters/
- Transitioning guide: https://docs.adapterhub.ml/transitioning.html
- Release notes: https://github.com/adapter-hub/adapters/releases/tag/v0.1.0
- The old codebase has been moved to: https://github.com/adapter-hub/adapter-transformers-legacy

### EMNLP 2023 system demonstration:

_Adapters_ has been presented as a system demonstration paper at EMNLP 2023.

- Paper: https://aclanthology.org/2023.emnlp-demo.13.pdf
- Demo materials: https://adapterhub.ml/emnlp2023/

**Comment by stefan-it** [comment]
Hi @calpt and the Adapters-team,

many thanks for your great work on the Adapters library!

I also really liked the blog post, and I am going to try it soon for NER 🤗 

One minor feedback to the blog post: Figure 2 states that test set of English CoNLL is used, but the results should be from development set :)

**Reply by calpt** [reply]
Thanks for your interest in our work, glad you like it!

Good catch on the results figure in our blog post! The CoNLL column is corrected in our paper (Table 3) but it seems we missed updating it in the post. Will do so now :)
